tasks/Preprocessing/CollectParentsKmerStats.wdl

```diff
@@ -113,7 +113,7 @@ task ParentalReadsRepartitionAndMerylConfigure {
     }
 
     String debug_option = if (run_with_debug) then "-debug" else " "
-    String kmer_option = if (defined(kmerSize)) then ("-triobinK " + kmerSize) else " "
+    String kmer_option = if (defined(kmerSize)) then ("-triobinK " + select_first([kmerSize])) else " "
```

Comment:
This is due to miniwdl not liking concatenating String with any optional variable. So here we make kmerSize none-optional using select_first.